test/test_docker.rb

```diff
@@ -7,51 +7,46 @@
       assert_equal 0, exit_code
     end
   end
+
   test "can run command in docker" do
     docker_test_command="grep docker /proc/1/cgroup"
     cassette(:get_commits, :record => :new_episodes) do
       cassette(:run_docker, :record => :all) do
 
         PRW.try_merge
         unless ENV['_system_version']
-          output, exit_code= PRW.run_command_in_shell(docker_test_command)
-          assert_equal "", output
```

Comment:
If there's an error condition, there'll be output, and it'll catch that in the test.